libtsuba/src/tsuba.cpp

```diff
@@ -220,6 +220,7 @@ tsuba::CreateSrcDestFromViewsForCopy(
     auto rdg_manifest_res =
         tsuba::RDGManifest::Make(uri, rdg_view.view_type, version);
     if (!rdg_manifest_res) {
+      KATANA_LOG_WARN("not a valid manifest file, uri: {}", uri.string());
```

Comment:
Would we be grabbing a Tracer object here instead of WARN?  I'm really not sure.

Reply:
I didn't think this was appropriate for a Tracer print so I stuck with a warning log.